saleor/plugins/base_plugin.py

```diff
@@ -120,6 +122,10 @@ def calculate_order_shipping(
     def calculate_checkout_line_total(
         self,
         checkout_line: "CheckoutLine",
+        variant: "ProductVariant",
```

Comment:
The changes in this file will break compatibility with the custom plugins - as the interface is changed. 
Is it required?  How do we want to handle this case (break compatibility of the custom plugins)?

Reply:
Bad news - I need to change signatures of most plugins related to checkout as now they do A LOT of queries, while they should receive already prefetched data. I don't know yet how we want to handle compatibility.